src/index.ts

```diff
@@ -18,13 +19,10 @@ const renderUsers = async (config: ApiConfig) => {
 };
 
 const app = () => {
-  const config = (window as any).__CONFIG__;
-  delete (window as any).__CONFIG__;
```

Comment:
Is there any reason to move this code into src/ioc/index.ts file ?

you can register the config right here:

ioc.register('config',  config);